spec/factories/vehicle_factory.rb

```diff
@@ -5,5 +5,6 @@
     loan_amount_outstanding { Faker::Number.decimal(l_digits: 4, r_digits: 2) }
     date_of_purchase { Faker::Date.between(from: 6.years.ago, to: 2.months.ago) }
     in_regular_use { Faker::Boolean.boolean }
+    subject_matter_of_dispute { Faker::Boolean.boolean }
```

Comment:
These feel too important to be set to a random value - should it not be false by default?